src/psycopmlutils/utils.py

```diff
@@ -27,11 +29,46 @@ def generate_feature_colname(
         interval_days (int): Fills out "_within_{interval_days}" in the col name.
         resolve_multiple (str): Name of the resolve_multiple strategy.
         fallback (str): Values used for fallback.
+        loader_kwargs (dict, optional): Loader kwargs. Defaults to None.
 
     Returns:
         str: _description_
     """
-    return f"{prefix}_{out_col_name}_within_{interval_days}_days_{resolve_multiple}_fallback_{fallback}"
+    if isinstance(out_col_name, str):
+        col_name = f"{prefix}_{out_col_name}_within_{interval_days}_days_{resolve_multiple}_fallback_{fallback}"
+        if loader_kwargs:
+            col_name = f"{col_name}_{format_dict_for_printing(loader_kwargs)}"
+    elif isinstance(out_col_name, list):
+        col_name = [
+            f"{prefix}_{col}_within_{interval_days}_days_{resolve_multiple}_fallback_{fallback}"
+            for col in out_col_name
+        ]
+        if loader_kwargs:
+            col_name = [
+                f"{col}_{format_dict_for_printing(loader_kwargs)}" for col in col_name
+            ]
```

Comment:
This assumes that the function is passed either its own kwargs or a `loader_kwargs` dict. 

The user might pass the `fallback` kwarg and not see it in `col_name`.

I see a couple of possible solutions:
1. If we want to make this assumption, we should probably check it. 
2. Use the base form (the one if no `loader_kwargs` dict passed), and then append formatted `loader_kwargs` to the colname
3. Combine passed kwargs into `loader_kwargs`, then use `loader_kwargs` for colname

I prefer 2., since `loader_kwargs` seems like a special case of passing a single column.

Reply:
No, this just adds the `loader_kwargs` at the end of the string (after `fallback` etc) if `loader_kwargs` is specified. This already does number 2 ;)